test_conformance/extensions/cl_khr_external_semaphore/main.cpp

```diff
@@ -26,6 +27,7 @@ test_definition test_list[] = {
     ADD_TEST(external_semaphores_cross_queues_io2),
     ADD_TEST(external_semaphores_multi_signal),
     ADD_TEST(external_semaphores_multi_wait),
+    ADD_TEST_VERSION(external_semaphores_import_export_fd, Version(1, 2)),
```

Comment:
I know this line just moved over from the previous test, but it's inconsistent with the other external semaphore tests.

Options:
1. Add the OpenCL 1.2 check to the other external semaphore tests.  This is probably the right thing to do because the spec says that semaphores require OpenCL 1.2 (though I forget why).
2. Remove the OpenCL 1.2 version check from this test.

Reply:
Corrected